src/Platform/Microsoft.Testing.Platform/CommandLine/PlatformCommandLineProvider.cs

```diff
@@ -34,6 +34,13 @@ internal sealed class PlatformCommandLineProvider : ICommandLineOptionsProvider
     public const string TestHostControllerPIDOptionKey = "internal-testhostcontroller-pid";
     public const string ExitOnProcessExitOptionKey = "exit-on-process-exit";
 
+    public const string VerbosityTrace = "Trace";
+    public const string VerbosityDebug = "Debug";
+    public const string VerbosityInformation = "Information";
+    public const string VerbosityWarning = "Warning";
+    public const string VerbosityError = "Error";
+    public const string VerbosityCritical = "Critical";
```

Comment:
Nit: Would it be possible to use an array of strings here? The check below seems easier that way, and it seems easier to later print that to help etc.

Nit, because you are just moving the code that was there before, and the set of options here seems pretty final.